src/scss/components/_list.scss

```diff
@@ -55,29 +52,37 @@
     display: block;
 
     .list__item {
-      display: block;
       width: auto;
       float: none;
+    }
 
-      &::before {
-        margin-right: 2px;
-      }
+    &[class*='col-'] {
+      display: block;
     }
   }
 
   &__item {
-    @include flex-shrink(0);
-    @include flexbox;
-
-    padding: 8px 0;
```

Comment:
@iondrimba I was testing here and the problem continue. My suggestion to avoid this is:

```CSS

.list__item a {
  transition: inherit;
  padding: 5px 5px 5px 0;
  width: calc(100% - 8px);
}
```

Calc will remove the translate transform and all will be fine :)